### ISSUE TITLE
Saving user details with a date of birth present crashes with PostgreSql

### ISSUE THREAD

**Issue by CrispinF** [cloudscribe/cloudscribe#959]
An exception occurred in the database while saving changes for context type '"cloudscribe.Core.Storage.EFCore.PostgreSql.CoreDbContext"'." ""Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details. ---> System.InvalidCastException: Cannot write DateTime with Kind=Unspecified to PostgreSQL type 'timestamp with time zone', only UTC is supported.

**Comment by JimKerslake** [comment]
The PR for this one includes a migration that doesn't actually do anything at all...?

**Comment by markidox** [comment]
No. Because I'd made a change to the db context I did run the migration but it just created an empty one...I then wasn't sure whether reversing that out was a good idea or not...

**Comment by JimKerslake** [comment]
That conversion was actually leading to the value (if it was a summer birthday) getting stored in the Db with one hour subtracted ... and then the UI datepicker failing to accommodate that back in the other direction when re-editing.  Hence the birthday seemed to shift a day forwards.

Painful stuff these dates - I'll try implementing my own fix.  Should really be a DateOnly column...
